src/Rules/Resources/Descriptions.resx

```diff
@@ -307,7 +307,7 @@
     <value>The BoundingRectangle property must not be defined as [0,0,0,0]</value>
   </data>
   <data name="BoundingRectangleNotNull" xml:space="preserve">
-    <value>An onscreen element must not have a null BoundingRectangle property.</value>
+    <value>An on-screen element must not have a null BoundingRectangle property.</value>
```

Comment:
There should be a corresponding change in `Descriptions.Designer.cs`. These are automatically kept in sync if edited via the IDE, but require manual synchronization if the files are edited outside the IDE.